spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/ChannelNode.java

```diff
@@ -21,21 +21,18 @@
 /**
  * @author Andy Clement
  * @author David Turanski
+ * @author Ilayaperumal Gopinathan
  */
 public class ChannelNode extends AstNode {
 
-	private ChannelType channelType;
+	private String nameComponent;
 
-	private List<String> nameComponents;
+	private List<String> labelComponents;
```

Comment:
Not all channel references are related to "labels", which are a tap-only concept, are they?